atomicapp/providers/openshift.py

```diff
@@ -371,13 +395,19 @@ def undeploy(self):
         logger.debug("Starting undeploy")
         delete_artifacts = []
         for kind, objects in self.openshift_artifacts.iteritems():
-            delete_artifacts.extend(objects)
+            # Add DCs to beginning of list so they are deleted first.
```

Comment:
"beginning of the list" 
I think we should have "deployment config" instead of the DC acronym.